apps/remix-ide/src/app/plugins/vyper-compilation-details.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react'
+import { ViewPlugin } from '@remixproject/engine-web'
+import {PluginViewWrapper} from '@remix-ui/helper'
+import { RemixAppManager } from '../../remixAppManager'
+import { RemixUiVyperCompileDetails } from '@remix-ui/vyper-compile-details'
+import { ThemeKeys, ThemeObject } from '@microlink/react-json-view'
+//@ts-ignore
+const _paq = (window._paq = window._paq || [])
+
+const profile = {
+  name: 'vyperCompilationDetails',
+  displayName: 'Vyper Compile Details',
+  description: 'Displays details from vyper compiler',
+  location: 'mainPanel',
+  methods: ['showDetails'],
+  events: []
+}
+
+export class VyperCompilationDetailsPlugin extends ViewPlugin {
```

Comment:
This looks a bit strange that we aren't re using the solidity comp details plugin

Reply:
reusing the solidity comp details is the ultimate plan. Sadly this hasn't been merged in time.